aten/src/ATen/native/quantized/cpu/qbatch_norm.cpp

```diff
@@ -252,6 +276,8 @@ Tensor quantized_batch_norm(
 }
 
 TORCH_LIBRARY_IMPL(quantized, QuantizedCPU, m) {
+  m.impl("batch_norm",        q_batch_norm_impl<false>);
```

Comment:
is this function pointing to the 2d/3d implementations, but not supporting the 1d implementation?  Not sure if that's super intuitive.  Can we add a comment on why this is needed?

Reply:
yeah, ideally this should support 1d as well, but there is no support for 1d atm. sure I'll add a comment